### ISSUE TITLE
alpha1: `fmt` with single-liners doesn't work

### ISSUE THREAD

**Issue by mitchellh** [hashicorp/terraform#19154]
### Terraform Version
```
Terraform v0.12.0-alpha1
```

### Terraform Configuration Files
```hcl
variable "region" { default = "us-east-1" }
```

### Expected Behavior
I would expect this to already be properly formatted.

### Actual Behavior
It turns it into this:

```hcl
variable "region" {default = "us-east-1"}
```

### Steps to Reproduce
Save file above, run `fmt`

### Additional Context
WIth 0.11, `fmt` with a single liner broke this up into multiple lines automatically. It appears at some point we started discouraging single line and that's okay, but 0.12 behaves very differently!

**Comment by apparentlymart** [comment]
Fixed in #19664.

It seems pointless to allow it if the formatter is going to undo it, so the formatter will instead allow the user to decide whether to use the single-line form and add the spaces around the braces if so.

**Comment by mitchellh** [comment]
I like that better. 😄 👍
